doc/manual/strings.rst

```diff
@@ -230,12 +230,12 @@ a normal value:
 Using an index less than 1 or greater than ``end`` raises an error::
 
     julia> str[0]
-    ERROR: BoundsError()
-     in getindex at /Users/sabae/src/julia/usr/lib/julia/sys.dylib (repeats 2 times)
+    ERROR: BoundsError: attempt to access 14-element Array{UInt8,1} at index [0]
+     in next(::String, ::Int64) at ./unicode/utf8.jl:63
+     in getindex(::String, ::Int64) at ./strings/basic.jl:58
 
     julia> str[end+1]
-    ERROR: BoundsError()
-     in getindex at /Users/sabae/src/julia/usr/lib/julia/sys.dylib (repeats 2 times)
+    ERROR: MethodError: no method matching endof(::Base.#std)
```

Comment:
What's this error message? I'm not on the very latest git master right now, but I don't see this.